packages/flutter/lib/src/material/drawer_item.dart

```diff
@@ -26,8 +36,13 @@ class DrawerItem extends StatelessWidget {
   /// The widget below this widget in the tree.
   final Widget child;
 
+  /// Callend when the user taps this drawer item.
   final VoidCallback onPressed;
 
+  /// Whether this drawer item is currently selected.
+  ///
+  /// The currently selected item is highlighted with additional color to
```

Comment:
"additional color" is an odd turn of phrase... did you perhaps mean "a different color"? Or is the color increased in saturation or something?

Maybe just leave out "with additional color" entirely since "highlighted" probably won't cause confusion.